stdlib/public/SDK/SceneKit/SceneKit.swift

```diff
@@ -29,9 +29,9 @@ extension SCNVector3 {
     self.z = SCNFloat(z)
   }
   public init(_ x: CGFloat, _ y: CGFloat, _ z: CGFloat) {
-    self.x = SCNFloat(x)
-    self.y = SCNFloat(y)
-    self.z = SCNFloat(z)
+    self.x = SCNFloat(x as NSNumber)
+    self.y = SCNFloat(y as NSNumber)
+    self.z = SCNFloat(z as NSNumber)
```

Comment:
Agreed - debugging in, though, we've definitely been introducing this conversion implicitly. I've filed SR-1432 to track my investigation of what's been going on.